app/konnect/updates.md

```diff
@@ -3,11 +3,117 @@ title: Konnect Cloud Updates
+
+: Existing organizations will be automatically upgraded to the new {{site.konnect_short_name}} environment over the next 4-6 weeks. We will be contacting your organization administrator with more details on the upgrade process.
+
+: You can continue using your `konnect.konghq.com` account and wait until we migrate it, or create a new account and [manually migrate configuration to the new account](/konnect/getting-started/import).
+
+: The minimum supported {{site.base_gateway}} version in this environment is 2.5.0.1.
+
+**Runtime groups**
+: {{site.konnect_short_name}} now manages all runtime configuration through runtime groups, which provide the ability to securely isolate configuration for sets of runtime instances. Essentially,  this gives you multiple SaaS-managed control planes in one {{site.konnect_short_name}} organization.
+
+: Every organization starts with one `default` runtime group. Additional custom runtime groups are an Enterprise-only feature.
+
+: Learn more about [runtime groups](/konnect/runtime-manager/runtime-groups), then read up on how to manage them with the [{{site.konnect_short_name}} UI](/konnect/runtime-manager/runtime-groups/manage) or with [decK](/konnect/runtime-manager/runtime-groups/declarative-config).
+
+: With runtime groups come a few other changes to runtime management for all organizations:
+  * Improvements with certificate rotation and management:
+    * When setting up runtime instances through the {{site.konnect_short_name}} UI, certificates are generated in the browser and pushed to the {{site.konnect_short_name}} API.
+    * {{site.konnect_short_name}} doesn't require CA certificates for runtime instances anymore.
+    * The validity period for runtime instance certificates has been extended from six months ten years.
+  * Reworked Gateway configuration UI:
+    * The Shared Config menu is now part of Runtime Manager. Manage your Gateway services, routes, plugins, upstreams, SNIs, and certificates through a runtime group, alongside all of the runtime instances in that group.
+  * You can use [labels for categorizing runtime groups](/konnect/runtime-manager/runtime-groups/manage).
+  Labels are key:value pairs, and are helpful for organizing, searching, and filtering subsets of {{site.konnect_short_name}} entities.
+
+**Teams and roles**
+: You can now manage {{site.konnect_short_name}} authorization with [teams and roles](/konnect/org-management/teams-and-roles).
+Existing RBAC roles have been converted to [predefined teams](/konnect/org-management/teams-and-roles/teams-reference), which are available for all {{site.konnect_short_name}} organizations.
+
+: Invite users and add them to these teams to manage user access. You can also assign individual roles per user.
```

Comment:
```suggestion
: To manage user access invite users to {{site.konnect_short_name}} and add them to teams, or assign individual roles to a user. 
```
the "manage user access" part felt odd where it was. I read it as conditional, so I figured it might sound better in the front.